lib/CL/pocl_cl.h

```diff
@@ -1174,6 +1184,11 @@ struct _cl_device_id {
      runtime/driver allocate the local space. */
   int device_alloca_locals;
 
+  /* Optional property. If the device uses stack and has stack size limitation
+   * (e.g. it does not convert all private memory to registers/spills),
+   * it should set up this property*/
+  size_t work_group_stack_size;
```

Comment:
Perhaps should document what indicates the optionality (when this property is zero? When some macro is set?).